v4/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/newrelic/go-agent/v4
 
-go 1.7
+go 1.13
```

Comment:
OpenTelemetry-Go only supports go versions 1.13 and above: https://github.com/open-telemetry/opentelemetry-go/blob/master/.circleci/config.yml#L3-L10